evap/static/css/evap.scss

```diff
@@ -696,10 +696,17 @@ select[multiple] {
     min-height: 40px;
 }
 
+.warning-label {
+    color: $evap-dark-yellow;
+}
 .error-label {
     color: $evap-dark-red;
 }
 
+.warning-border {
+    border: 1px solid $evap-yellow;
+}
```

Comment:
please add an additional `outline-color: $evap-yellow;` for chrome

Reply:
:+1: